core/src/main/res/values/strings.xml

```diff
@@ -94,74 +94,74 @@
     <string name="reset">Reset</string>
     <string name="url_label">URL</string>
     <string name="support_funding_label">Support</string>
-    <string name="support_podcast">Support this Podcast</string>
+    <string name="support_podcast">Support this Creator</string>
     <string name="error_label">Error</string>
     <string name="error_msg_prefix">An error occurred:</string>
     <string name="refresh_label">Refresh</string>
-    <string name="external_storage_error_msg">No external storage is available. Please make sure that external storage is mounted so that the app can work properly.</string>
+    <string name="external_storage_error_msg">Please mount external storage so that the app access it first.</string>
```

Comment:
```suggestion
    <string name="external_storage_error_msg">Please mount a storage device so that the app access it first.</string>
```
(used elsewhere)